### PR TITLE
NXCM-3593, switch to EPL license

### PR DESCRIPTION
**Pull request by bdemers**
This needs to be merged manually, after running: mvn license:format -N
(running before the pull request would make it impossible to review)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -121,45 +121,14 @@
...
-              <phase>validate</phase>
-            </execution>
-            <execution>
-              <id>license-check-alv2</id>
-              <goals>
-                <goal>check</goal>
-              </goals>
-              <phase>validate</phase>
-              <configuration>
-                <header>header-alv2.txt</header>
```
Comment (bdemers) [inline pom.xml:144]: NOTE: this license now matches the Nexus license.

**Comment by velo** [comment]
Nice approach
+1
Just do not forget to update licenses =D

**Comment by nabcos** [comment]
+1

**Comment by adreghiciu** [comment]
+1. We may want to change license on capabilities plugin and p2/obr

**Comment by cstamas** [comment]
:beer:

**Comment by peterlynch** [comment]
:beer:

**Comment by mcculls** [comment]
+1

**Comment by bdemers** [comment]
merged manually

**Pull request closed without merging**